diff --git a/plugin/youcompleteme.vim b/plugin/youcompleteme.vim
index b6673bee..fd34190d 100644
--- a/plugin/youcompleteme.vim
+++ b/plugin/youcompleteme.vim
@@ -277,6 +277,13 @@ let g:ycm_java_jdtls_workspace_root_path =
 let g:ycm_python_binary_path =
       \ get( g:, 'ycm_python_binary_path', '' )
 
+" Use system Binary.
+let g:ycm_gopls_binary_path = 'gopls'
+let g:ycm_rls_binary_path = 'rls'
+let g:ycm_rustc_binary_path = 'rustc'
+let g:ycm_tsserver_binary_path= 'tsserver'
+let g:ycm_roslyn_binary_path = '/opt/omnisharp-roslyn/OmniSharp.exe'
+
 if has( 'vim_starting' ) " Loading at startup.
   " We defer loading until after VimEnter to allow the gui to fork (see
   " `:h gui-fork`) and avoid a deadlock situation, as explained here:
diff --git a/python/ycm/paths.py b/python/ycm/paths.py
index 1fa158c5..90870288 100644
--- a/python/ycm/paths.py
+++ b/python/ycm/paths.py
@@ -19,11 +19,11 @@ import os
 import sys
 import vim
 import re
+import ycmd
 
 # Can't import these from setup.py because it makes nosetests go crazy.
 DIR_OF_CURRENT_SCRIPT = os.path.dirname( os.path.abspath( __file__ ) )
-DIR_OF_YCMD = os.path.join( DIR_OF_CURRENT_SCRIPT, '..', '..', 'third_party',
-                            'ycmd' )
+DIR_OF_YCMD = os.path.normpath( os.path.join( ycmd.__file__, '..', '..' ) )
 WIN_PYTHON_PATH = os.path.join( sys.exec_prefix, 'python.exe' )
 PYTHON_BINARY_REGEX = re.compile(
   r'python(3(\.[6-9])?)?(.exe)?$', re.IGNORECASE )
