diff --git a/ycmd/utils.py b/ycmd/utils.py
index 302db395..1cd714b3 100644
--- a/ycmd/utils.py
+++ b/ycmd/utils.py
@@ -27,9 +27,8 @@ import time
 import threading
 
 LOGGER = logging.getLogger( 'ycmd' )
-ROOT_DIR = os.path.normpath( os.path.join( os.path.dirname( __file__ ), '..' ) )
-DIR_OF_THIRD_PARTY = os.path.join( ROOT_DIR, 'third_party' )
-LIBCLANG_DIR = os.path.join( DIR_OF_THIRD_PARTY, 'clang', 'lib' )
+ROOT_DIR = os.path.dirname( __file__ )
+LIBCLANG_DIR = '/usr/lib'
 if hasattr( os, 'add_dll_directory' ):
   os.add_dll_directory( LIBCLANG_DIR )
 
