diff --git a/src/editor/editor.pro b/src/editor/editor.pro
index 0625867..da4d47e 100644
--- a/src/editor/editor.pro
+++ b/src/editor/editor.pro
@@ -1,9 +1,13 @@
 QT += core gui widgets network svg
+QT += KSyntaxHighlighting
 
 TARGET = VTextEdit
 TEMPLATE = lib
 CONFIG += shared dll
 
+CONFIG += link_pkgconfig
+PKGCONFIG += hunspell
+
 TRANSLATIONS += data/translations/vtextedit_zh_CN.ts
 
 macx: {
@@ -12,15 +16,6 @@ macx: {
     FRAMEWORK_HEADERS.files = $$files(include/vtextedit/*.h)
     FRAMEWORK_HEADERS.path = Headers/vtextedit
     QMAKE_BUNDLE_DATA += FRAMEWORK_HEADERS
-
-    # Process VSyntaxHighlighting framework
-    sh_lib_name = VSyntaxHighlighting
-    sh_lib_dir = $${OUT_PWD}/../libs/syntax-highlighting
-    sh_lib_full_name = $${sh_lib_name}.framework/Versions/1/$${sh_lib_name}
-    lib_target = $${TARGET}.framework/Versions/1/$${TARGET}
-    QMAKE_POST_LINK = \
-        install_name_tool -add_rpath $${sh_lib_dir} $${lib_target} && \
-        install_name_tool -change $${sh_lib_full_name} @rpath/$${sh_lib_full_name} $${lib_target}
 }
 
 DEFINES += VTEXTEDIT_LIB
@@ -46,7 +41,6 @@ INCLUDEPATH += $$PWD
 INCLUDEPATH += $$PWD/include
 
 LIBS_FOLDER = $$PWD/../libs
-include($$LIBS_FOLDER/syntax-highlighting/syntax-highlighting_export.pri)
 include($$LIBS_FOLDER/katevi/katevi_export.pri)
 include($$LIBS_FOLDER/peg-markdown-highlight/peg-markdown-highlight_export.pri)
 include($$LIBS_FOLDER/sonnet/src/core/core_export.pri)
diff --git a/src/editor/texteditor/syntaxhighlighter.cpp b/src/editor/texteditor/syntaxhighlighter.cpp
index d761862..4ddfa11 100644
--- a/src/editor/texteditor/syntaxhighlighter.cpp
+++ b/src/editor/texteditor/syntaxhighlighter.cpp
@@ -35,21 +35,6 @@ SyntaxHighlighter::SyntaxHighlighter(QTextDocument *p_doc,
         qDebug() << "use definition" << def.name() << "to highlight for syntax" << p_syntax;
         setDefinition(def);
     }
-
-    KSyntaxHighlighting::Theme th;
-    if (!p_theme.isEmpty()) {
-        // Check if it is a file path.
-        if (p_theme.contains(QRegularExpression(QStringLiteral("[\\\\/]")))) {
-            th = repository()->themeFromFile(p_theme);
-        } else {
-            th = repository()->theme(p_theme);
-        }
-    }
-    if (!th.isValid()) {
-        th = repository()->defaultTheme();
-    }
-    setTheme(th);
-    qDebug() << "use syntax highlighter theme" << th.name() << p_theme;
 }
 
 void SyntaxHighlighter::highlightBlock(const QString &p_text)
diff --git a/src/libs/libs.pro b/src/libs/libs.pro
index 870befe..9f362cb 100644
--- a/src/libs/libs.pro
+++ b/src/libs/libs.pro
@@ -1,7 +1,6 @@
 TEMPLATE = subdirs
 
 SUBDIRS += \
-    syntax-highlighting \
     katevi \
     peg-markdown-highlight \
     sonnet-data \
